Sources/LayoutSpecBuilders/Layout/CornerLayout.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CornerLayout.swift
+//  TextureSwiftSupport
+//
+//  Created by arya.cia on 14/03/22.
+//  Copyright © 2022 muukii. All rights reserved.
+//
+
+import Foundation
+
+public struct CornerLayout<CornerContent, Content> : _ASLayoutElementType where CornerContent : _ASLayoutElementType, Content : _ASLayoutElementType {
+
+  public let content: Content
+  public let cornerContent: CornerContent
+  public let location: ASCornerLayoutLocation
+  
+  public init(
+    location: ASCornerLayoutLocation = .topRight,
+    child: () -> Content,
+    corner: () -> CornerContent
```

Comment:
Would you change this parameter like this?
I know other components are using closure to get their child. but I'm thinking those should be changed to use passing instances without closure.

```suggestion
    content: Content,
    cornerContent: CornerContent
```

Reply:
I don't mind changing them, but CMIIW, if I am about to pass the content directly, the syntax will look something like this
<img width="521" alt="Screen Shot 2022-03-15 at 08 41 49" src="https://user-images.githubusercontent.com/40857328/158289261-6988274a-1a12-45f2-b60d-8938e48b3c5c.png">

which in turn is pretty much the same as the original style of texture, isn't the closure that gives us the swiftUI syntax taste?

Reply:
For now, I changed my mind, it should not be using closure to get value and I'll change others according to this later.
Because 
- It makes complex what closure makes, returns or creates layout.
- Allocating closure might get some cost.
- Why SwiftUI uses closure is making ResultBuilder available.
- Otherwise it should not use closure. For instance, SwiftUI.AnyView uses not closure.

Reply:
that makes sense, thanks for the explanation. Changed here https://github.com/TextureCommunity/TextureSwiftSupport/pull/82/commits/462250732baca44a737bc73513612396ed7f1c99